Documentation/Signoffs/SteeringSystem.md

```diff
@@ -0,0 +1,115 @@
+STEERING WHEEL SUBSYSTEM
+------------------------
+### Function of the Subsystem
+The Steering Wheel subsytem shall recieve the input of the Nissan Leaf steering wheel into a sensor
+embedded into the steering column of the vehicle. The sensor will measure the steering angle and send
+the data into a local microcontroller which will interpret the data and relay it to the simulation.
+
+### Specifications & Constraints
+- Must measure angle of rotation of the steering wheel
+- Must use the Nissan Leaf OEM steering wheel as the input
+- Angle shall be no more than 960 degrees from the left boundary to the right boundary (2 and 2/3 rotations)
+- Error of the angle must be within 1% of the true steering angle
+- Must not lose steering position when power is cycled
+- System must be small enough to fit in the area of the vehicle without interfering with the driver or other subsytems
+
+The simulation requires an input for the steering angle of the vehicle. In the Nissan Leaf, the steering
+is limited to 2 and 2/3 rotations from the center point to each clockwise and counter-clockwise direction.
+Therefore, the angle measured must be limited to 960 degrees in each direction. Due to the broader impact 
+accuracy constraint, the error of the angle sensed must be within 1% of the true angle to prevent false
+representation of the steering.
+
+### Schematic
+##### 1. Model
+
+
+![image](https://user-images.githubusercontent.com/100802413/200971670-3beca395-ed8c-400a-b166-59c4ad6b9142.png)
+
+_2D-Model of Gear and Encoder Connection_
```

Comment:
Encoder has nowhere to mount and does not have a way to connect to the gears. Not buildable.